web-client/iron-remote-gui/src/services/wasm-bridge.service.ts

```diff
@@ -258,35 +259,84 @@ export class WasmBridgeService {
         return onClipboardChangedPromise();
     }
 
+    setKeyboardUnicodeMode(use_unicode: boolean) {
+        this.keyboardUnicodeMode = use_unicode;
+    }
+
     private releaseAllInputs() {
         this.session?.release_all_inputs();
     }
 
+    private supportsUnicodeKeyboardShortcuts(): boolean {
+        if (this.session?.supports_unicode_keyboard_shortcuts) {
+            return this.session?.supports_unicode_keyboard_shortcuts();
```

Comment:
question: Maybe this should be called once and cached? Otherwise we have the JavaScript <-> WASM FFI cost.